### DISCUSSION TITLE
Right Sidebar position fixed in container at 1200px

### DISCUSSION THREAD

**Discussion by Ludinator** [templaza/astroid-framework#343] [General]
Hallo. Ich habe im Astroid Template das Site Layout auf "Wide" gestellt und Theme Width auf "1200px". Nun möchte ich das meine rechte Sidebar mit Modulen innerhalb dieser 1200 px erscheint. Wie kann ich das erreichen?
Aktuell wird der gesamte Content über die ganze Bildschirmbreite gezeigt, sobald ich ein Modul auf der Position "right" veröffentliche.

Das "Boxed" Layout möchte ich nicht nutzen das Header und Footer über die ganze Breite gehen sollen.

Danke. Gruß Frank

----------------------

Hello. In the Astroid Template I set the Site Layout to "Wide" and Theme Width to "1200px". Now I want my right sidebar with modules to appear within that 1200px. How can I achieve this?
Currently, the entire content is shown across the entire width of the screen as soon as I publish a module in the "right" position.

I don't want to use the "boxed" layout because the header and footer should cover the entire width.

Thanks. Greeting Frank

**Comment by deltapapa01** [comment]
DE:
Moin Frank,
schau mal hier, ich hatte da ein ähnliches Problem, vielleicht hilft dir das irgendwie weiter: https://github.com/templaza/astroid-framework/discussions/284
Ich persönlich baue alle meine Seiten so um, das ich weder links noch rechts Module nutze, sondern diese unter den Content anordne. Hintergrund: Die Nutzung auf dem Handy hat die auf breiten Desktops weit überholt, und da ist die Anzeige in der Breite immer ein Problem. Wenn du mal hier schaust: https://taekwondo-husum.de/, diese Module waren bei dem vorherigen Template (JSN Epic) links und rechts, jetzt sind sind unter dem Content. Der Nutzer sieht auf dem Handy also immer erst den Header, dann Content und dann evtl Zusatzinfos in den Boxen. Nur mal als Idee....
<hr>
EN:
Hello Frank,
look here, I had a similar problem, maybe this helps you somehow: https://github.com/templaza/astroid-framework/discussions/284
I personally rebuild all my pages so that I use neither left nor right modules, but arrange them below the content. Background: The use on cell phones has overtaken the use on wide desktops, and there the display in the width is always a problem. If you look here: https://taekwondo-husum.de/, these modules were in the previous template (JSN Epic) on the left and right, now are under the content. So the user sees on the mobile first the header, then content and then possibly additional info in the boxes. Just an idea....

WBR,
deltapapa

**Reply by Ludinator** [reply]
Hallo deltapapa01, deinen Vorschlag hatte ich auch schon mal in Betracht gezogen und habe es dann auch so gemacht. Mir ging es halt um immer wiederkehrende Module auf allen Seiten. Das wäre mit einer Sidebar praktisch. Aber ich arbeite mit einem Pagebulder und lade die Module weinfach darüber in den Content.

**Comment by Chacapamac** [comment]
@Ludinator 
Just tell me if [discussion ](https://github.com/templaza/astroid-framework/discussions/284) solved your problem?

**Good Day!**
